osf/models/files.py

```diff
@@ -331,25 +330,15 @@ def touch(self, auth_header, revision=None, **kwargs):
         # TODO Switch back to head requests
         # return self.update(revision, json.loads(resp.headers['x-waterbutler-metadata']))
 
-    def get_page_counter_count(self, count_type, version=None):
-        """Assembles a string to retrieve the correct file data from the pagecounter collection,
-        then calls get_basic_counters to retrieve the total count. Limit to version if specified.
-        """
-        parts = [count_type, self.target._id, self._id]
-        if version is not None:
-            parts.append(version)
-        page = ':'.join([format(part) for part in parts])
-        _, count = get_basic_counters(page)
-
-        return count or 0
-
     def get_download_count(self, version=None):
         """Pull the download count from the pagecounter collection"""
-        return self.get_page_counter_count('download', version=version)
+        from osf.models import PageCounter
+        return PageCounter.get_file_downloads(self._id, version)
 
     def get_view_count(self, version=None):
         """Pull the mfr view count from the pagecounter collection"""
-        return self.get_page_counter_count('view', version=version)
+        from osf.models import PageCounter
+        return PageCounter.get_file_views(self._id)
```

Comment:
drive-by: should `version` be passed here?

Reply:
👍